composer.json

```diff
@@ -51,7 +51,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "0.8-dev"
+            "dev-master": "1.0-dev"
```

Comment:
Based on the discussion on #80 I assume this would be new major version. This could certainly be left at 0.8 or be bumped to any other number. That's a hugely minor detail.